R/LongitudinalModel.R

```diff
@@ -28,13 +28,19 @@ LongitudinalModel <- function(
     ...
 ) {
 
-    base_long <- StanModule(
-        x = "base/longitudinal.stan"
+    base_stan <- paste0(
+        read_stan("base/longitudinal.stan"),
+        collapse = "\n"
```

Comment:
side question: is this pasting/collapsing really needed? I would have thought this kind of stuff is handled in `read_stan` & co.?